src/common/kube-helpers.ts

```diff
@@ -0,0 +1,162 @@
+import { app, remote } from "electron";
+import { KubeConfig, V1Node, V1Pod } from "@kubernetes/client-node"
+import { ensureDirSync, readFile, writeFileSync } from "fs-extra";
+import path from "path"
+import os from "os"
+import yaml from "js-yaml"
+import logger from "../main/logger";
+
+function resolveTilde(filePath: string) {
+  if (filePath[0] === "~" && (filePath[1] === "/" || filePath.length === 1)) {
+    return filePath.replace("~", os.homedir());
+  }
+  return filePath;
```

Comment:
Should this handle `~fred` => `/home/fred` (or `/Users/fred`)?

Reply:
This handles `~/...` => `/home/fred/...` and `~` => `/home/fred/`. A unit test would not hurt.